load.php

```diff
@@ -48,5 +48,10 @@
 		define( 'DB_NAME', 'test' );
 		define( 'EP_INDEX_PREFIX', 'tests_' );
 		define( 'WP_CACHE_KEY_SALT', 'codecept' );
+
+		// Support acceptance testing delayed execution. See AcceptanceTester::preLoadConfig.
+		if ( file_exists( Altis\ROOT_DIR . '/tests/include.php' ) ) {
+			include_once( Altis\ROOT_DIR . '/tests/include.php' );
```

Comment:
Could we call this `load.php` for consistency?

Reply:
Yeah, makes sense.

Reply:
`bootstrap.php` would be more consistent IMO, at least with PHPUnit as this is for setting up tests. But let's not bikeshed.

One thing that springs to mind is that the `tests` directory might already have PHPUnit / WPUnit tests in it. Are there any common file names people might already be using in that directory that we should avoid colliding with? FWIW I think `load.php` is very likely safe, but another option could be to prefix it like `acceptance-bootstrap.php` for instance.

Reply:
It's not really bootstrapping the process, so I'm not keen on using `bootstrap.php` for it, `include.php` seemed appropriate at first, but `load.php` may be more more consistent.